bin/quickpkg

```diff
@@ -43,6 +46,19 @@ def quickpkg_atom(options, infos, arg, eout):
 	fix_metadata_keys = ["PF", "CATEGORY"]
 
 	try:
+		binpkg_compression = settings.get("BINPKG_COMPRESSION", "bzip2")
+		compression = _compressors[binpkg_compression]
+	except KeyError as e:
+		eout.eerror("Unsupported compression method: %s" % e.args[0])
+		del e
+		return
+
+	if subprocess.call(["type", compression["decompress"]], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE) != 0:
```

Comment:
Normally we use portage.process.find_binary() for this, because calling a shell is overkill (and python2.7 does not have shutil.which()).